wsgi_kerberos.py

```diff
@@ -156,12 +162,13 @@ def _authenticate(self, client_token):
                     user = kerberos.authGSSServerUserName(state)
                 elif rc == kerberos.AUTH_GSS_CONTINUE:
                     server_token = kerberos.authGSSServerResponse(state)
-        except kerberos.GSSError:
-            pass
+        except kerberos.GSSError as e:
+            exc = e
+            LOG.exception('Unhandled GSSError')
```

Comment:
(As a reminder, `log.exception` provides the exception info in the log message automatically: https://docs.python.org/3/library/logging.html#logging.Logger.exception)

Reply:
Is the behavior the same in python2? I assume the answer is yes.

The only potential issue I see for this is that we’re going to be logging a full stack trace when this occurs, whereas in the past it was silent. Maybe we should make this opt-in?

Reply:
> Is the behavior the same in python2? I assume the answer is yes.
[Yes](https://docs.python.org/2/library/logging.html#logging.Logger.exception). 

> The only potential issue I see for this is that we’re going to be logging a full stack trace when this occurs, whereas in the past it was silent. Maybe we should make this opt-in?

In the latest revision, I've added a `debug` param to the constructor that defaults to False. The full traceback is now only logged when this is set to True, otherwise just the exception itself is logged (with equivalent level, namely "error").